src/components/DonationItemsTable/DonationItemsTable.jsx

```diff
@@ -39,31 +46,37 @@ const DonationItemsTable = () => {
       }
     };
 
+    const fetchNotifications = async () => {
+      const response = await backend.get('/notification/0');
+      setNotification(response.data);
+    }
     checkIsAdmin();
-  });
+    fetchNotifications();
+  }, [backend]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'isAdmin' and 'navigate'. Either include them or remove the dependency array.